tests/doghouse/test_snapshot.py

```
@@ -0,0 +1,100 @@
+        blockers=[b2],
+    )
+    assert not s1.is_equivalent_to(s2)
+
+
+def test_not_equivalent_severity_change():
+    b1 = Blocker(id="t1", type=BlockerType.NOT_APPROVED, message="review",
+                 severity=BlockerSeverity.BLOCKER)
+    b2 = Blocker(id="t1", type=BlockerType.NOT_APPROVED, message="review",
+                 severity=BlockerSeverity.WARNING)
+    s1 = Snapshot(
+        timestamp=datetime.datetime(2026, 1, 1, tzinfo=datetime.timezone.utc),
```

Ruff (0.15.7)

[warning] Use `datetime.UTC` alias

Convert to `datetime.UTC` alias

(UP017)